pyomo/repn/tests/baron/test_baron_comparison.py

```diff
@@ -30,52 +41,47 @@ def pyomo(self, cmd):
         output = main.main(['convert', '--logging=quiet', '-c']+cmd)
         return output
 
-class BaselineTests(Tests):
-    def __init__(self, *args, **kwds):
-        Tests.__init__(self, *args, **kwds)
-
-#
-#The following test generates an BAR file for the test case
-#and checks that it matches the current pyomo baseline BAR file
-#
-@unittest.nottest
-def barwriter_baseline_test(self, name):
-    baseline = currdir+name+'.pyomo.bar'
-    output = currdir+name+'.test.bar'
-    if not os.path.exists(baseline):
-        self.skipTest("baseline file (%s) not found" % (baseline,))
-
-    if os.path.exists(datadir+name+'_testCase.py'):
-        testDir = datadir
-    else:
-        testDir = currdir
-    testCase = testDir+name+'_testCase.py'
-
-    if os.path.exists(testDir+name+'.dat'):
-        self.pyomo(['--output='+output,
-                    testCase,
-                    testDir+name+'.dat'])
-    else:
-        self.pyomo(['--output='+output,
-                    testCase])
-
-    # Check that the pyomo BAR file matches its own baseline
-    self.assertFileEqualsBaseline(
-        output, baseline, tolerance=(1e-7, False))
-
-
-class ASLTests(Tests):
 
+class BaselineTests(Tests):
     def __init__(self, *args, **kwds):
         Tests.__init__(self, *args, **kwds)
 
+    #
+    #The following test generates an BAR file for the test case
+    #and checks that it matches the current pyomo baseline BAR file
+    #
+    @parameterized.parameterized.expand(input=names)
+    def barwriter_baseline_test(self, name):
+        baseline = currdir+name+'.pyomo.bar'
+        output = currdir+name+'.test.bar'
+        if not os.path.exists(baseline):
+            self.skipTest("baseline file (%s) not found" % (baseline,))
+
+        if os.path.exists(datadir+name+'_testCase.py'):
+            testDir = datadir
+        else:
+            testDir = currdir
+        testCase = testDir+name+'_testCase.py'
+
+        if os.path.exists(testDir+name+'.dat'):
+            self.pyomo(['--output='+output,
+                        testCase,
+                        testDir+name+'.dat'])
+        else:
+            self.pyomo(['--output='+output,
+                        testCase])
+
+        # Check that the pyomo BAR file matches its own baseline
+        with open(output, 'r') as f1, open(baseline, 'r') as f2:
+            f1_contents = list(filter(None, f1.read().split()))
+            f2_contents = list(filter(None, f2.read().split()))
+            for item1, item2 in zip(f1_contents, f2_contents):
```

Comment:
This should probably use `itertools.zip_longest` so that differing length files cause errors.